numpy/ma/extras.py

```diff
@@ -1327,7 +1385,7 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
         return flag[indx][rev_idx]
 
 
-def isin(element, test_elements, assume_unique=False, invert=False):
+def isin(ar1, ar2, assume_unique=False, invert=False):
```

Comment:
Why did you change parameter names? I think `element, test_elements` matches `np.isin`.